public/app/features/plugins/admin/components/PluginDetailsBody.tsx

```diff
@@ -49,6 +69,24 @@ export function PluginDetailsBody({ plugin, queryParams, pageId }: Props): JSX.E
     );
   }
 
+  // Permissions will be returned in the iam field for installed plugins and in the details.iam field when fetching details from gcom
+  const permissions = plugin.iam?.permissions || plugin.details?.iam?.permissions;
+
+  if (
+    config.featureToggles.externalServiceAccounts &&
+    pageId === PluginTabIds.IAM &&
+    permissions &&
+    permissions.length > 0
```

Comment:
nit: can we put this logic in a variable so it's easier to scan the code?

Reply:
Changed as requested